### DISCUSSION TITLE
(Unraid) Trying to setup 'No Hardlinks' check but Qbit_Manage is telling me it can't open the files in my completed downloads folder.

### DISCUSSION THREAD

**Discussion by EmceeOrrie** [StuffAnThings/qbit_manage#504] [Q&A] [answered]
This is on Unraid. 

QBM tags the torrents with the trackers fine, but isn't able to determine hardlinks for some reason. I get this output every time:

[Activity Log](https://gist.github.com/EmceeOrrie/9b1e5e9307047ecfee000892091f53fd)

I have my share setup according to the Trash Guide so movies are downloaded to /data/torrents/movies/ and then 'moved' to /data/media/movies/. 

[Here are the folder permissions. ](https://gist.github.com/EmceeOrrie/acc005747c6ba4271307d8cd8628c3a8)

Both the media and torrents folders were set to root/root previously but updating them did not fix it.

[Here is my config.](https://gist.github.com/EmceeOrrie/cab705d6f1c3afa9723655e5df92d20d)

I should note that my BT_backup folder is in /appdata/qbittorrent/qBittorrent/ and I could not find a way to change it, nor would QBM accept that the path existed so I had to define:

`/bt_backup/ <-> /mnt/user/appdata/qbittorrent/qBittorrent/BT_backup/`

Not sure if that's related. 

Given my paths above, are these correct?

```
directory:
  root_dir: /data/torrents/
  torrents_dir: /bt_backup/
```

```
cat:
  movies: /data/torrents/movies
```

```
nohardlinks:
- movies
```

Any other ideas?

**Comment by bakerboy448** [comment]
Sounds like you failed to mount qbm to access to your downloads dir

**Reply by EmceeOrrie** [reply]
`root_dir: /data/torrents/`

Does this not do that? My downloads folders 'movies' and 'tv' are in there.

**Reply by bakerboy448** [reply, marked as answer]
File config has nothing at all to do with your container config of the environment and mounts

**Reply by EmceeOrrie** [reply]
OK, I seem to have fixed it. I didn't realize I had to change the root_dir container path from 'same/path/as/qbit' to the actual path. I figured it was something dumb.

Thanks!
